src/main/java/net/testudobank/MvcController.java

```diff
@@ -328,6 +342,140 @@ public String submitWithdraw(@ModelAttribute("user") User user) {
 
   }
 
+/**
+   * HTML GET request handler that serves the "transfer_form" page to the user.
+   * An empty `User` object is also added to the Model as an Attribute to store
+   * the user's transfer form input.
+   * 
+   * @param model
+   * @return "dispute_form" page
+   */
+  @GetMapping("/transfer")
+	public String showTransferForm(Model model) {
+    User user = new User();
+		model.addAttribute("user", user);
+		return "transfer_form";
+	}
+
+  /**
+   * HTML POST request handler for the Transfer Form page.
+   * 
+   * The same username+password handling from the login page is used.
+   * 
+   * If the password attempt is correct, the users transfer successfully goes through
+   * if it is a valid transfer. Both customers balances are properly updated.
+   * 
+   * If the password attempt is incorrect, the user is redirected to the "welcome" page.
+   * 
+   * @param user
+   * @return "account_info" page if login successful. Otherwise, redirect to "welcome" page.
+   */
+
+  @PostMapping("/transfer")
+  public String submitTransfer(@ModelAttribute("user") User user) {
+    String userID = user.getUsername();
+    String userPasswordAttempt = user.getPassword();
+    String userPassword = TestudoBankRepository.getCustomerPassword(jdbcTemplate, userID);
+    
+    /// Invalid Input/State Handling ///
+
+    // unsuccessful login
+    if (userPasswordAttempt.equals(userPassword) == false) {
+      return "welcome";
+    }
+
+    //If a customer already has too many reversals, or they try to send money to themselves, transfer fails.
+    int numOfReversals = TestudoBankRepository.getCustomerNumberOfReversals(jdbcTemplate, userID);
+    if (numOfReversals >= MAX_DISPUTES || user.getWhoToTransfer().equals(userID)) {
+      return "welcome";
+    }
+
+    // Negative transfer amount is not allowed
+    double userDepositAmt = user.getAmountToTransfer();
+    if (userDepositAmt < 0) {
+      return "welcome";
+    }
+
+    //checks to see the customer you are transfering to exists
+    TestudoBankRepository.doesCustomerExist(jdbcTemplate, user.getWhoToTransfer());
+    
+    double userWithdrawAmt = user.getAmountToTransfer();
+    int userWithdrawAmtInPennies = convertDollarsToPennies(userWithdrawAmt);
+
+    //Get the sender's balance in pennies.
+    int userBalanceInPennies = TestudoBankRepository.getCustomerBalanceInPennies(jdbcTemplate, userID);
```

Comment:
Unnecessary comment. Just rename the variable to `senderBalanceInPennies`, and remove the comment. The code should speak for itself.

Reply:
Still need to implement this.